oktapam/constants/descriptions/prefixes.go

```diff
@@ -4,4 +4,5 @@ const (
 	PrefixBeta = "Beta Feature:"
 	PrefixLEA  = "Limited Early Access Feature:"
 	PrefixDev  = "Feature under development:"
```

Comment:
Small nit, but if we tweak this, it'll better match the other options.
```suggestion
	PrefixDev  = "Under Development Feature :"
```